src/main/resources/locales/en/subscription-configure-list-confirmed.json

```diff
@@ -1,7 +1,7 @@
 {
     "title": "Court Subscription(s) refined",
-    "header": "Court Subscription(s) refined",
-    "subheader": "Your subscription(s) has been amended successfully",
+    "header": "List types updated",
+    "subheader": " ",
```

Comment:
subheader can be removed if it is not used. The njk will also need to be updated